pkg/virt-operator/resource/apply/apps.go

```diff
@@ -391,7 +391,9 @@ func (r *Reconciler) syncPodDisruptionBudgetForDeployment(deployment *appsv1.Dep
 }
 
 func getDesiredApiReplicas(clientset kubecli.KubevirtClient) (replicas int32, err error) {
-	nodeList, err := clientset.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+	nodeList, err := clientset.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{
+		LabelSelector: fmt.Sprintf("%s=%s", v1.NodeSchedulable, "true"),
```

Comment:
Would it make sense to use `k8s.io/apimachinery/pkg/labels` to construct the label selector? It's quite simple though.

Reply:
I did look at this but IMHO it's such a simple case that this is enough.